R/RestRserveAuthMiddleware.R

```diff
@@ -29,13 +29,13 @@
 #' @export
 #'
 #' @seealso
-#' [RestRserveMiddleware] [RestRserveApplication]
+#' [Middleware] [Application]
 #'
 #' @family AuthBackend
 #'
-RestRserveAuthMiddleware = R6::R6Class(
-  "RestRserveAuthMiddleware",
-  inherit = RestRserveMiddleware,
+AuthMiddleware = R6::R6Class(
+  "AuthMiddleware",
```

Comment:
Param name (`classname`).

Reply:
?

Reply:
Should be `classname = "AuthMiddleware"`